welcome/setup-dev-environment.md

```diff
@@ -62,11 +62,12 @@ Before we work on the Instant Answer codebase, we first need a development machi
 
 > Already have a Codio Account? Perfect, move on to the next step.
 
-1. Go to https://codio.com and click "**Sign Up**", at the top right corner.
-2. Click "**Sign Up via GitHub**".
-3. If you aren't already signed into GitHub, enter your GitHub login details and then click "**Sign In**".
-4. Click "**Authorize application**" to continue.
-5. In the new screen, enter the required details and click "**Create Account**".
+1. Go to https://codio.com and click "**Free Trial**", at the top right corner.
```

Comment:
This  would also be fine in my opinion.

Unfortunately, this will of course only hold as long as the path of the URL will stay "/signup".
The website is not consist, for example: the path to the login page is not just "/login" from the root but:  "/p/login" (https://codio.com/p/login). 

Alternatively the link in this case could also be: https://codio.com/signup/?type=INDIVIDUAL, which saves another step (step 2) and presents you immediately with the sign-up form.
This adds another subject to change part though (the query).